feathr_project/feathr/definition/config_helper.py

```diff
@@ -0,0 +1,193 @@
+from feathr.definition.dtype import *
+from feathr.registry.registry_utils import *
+from feathr.utils._file_utils import write_to_file
+from feathr.definition.anchor import FeatureAnchor
+from feathr.constants import *
+from feathr.definition.feature import Feature, FeatureType,FeatureBase
+from feathr.definition.feature_derivations import DerivedFeature
+from feathr.definition.repo_definitions import RepoDefinitions
+from feathr.definition.source import HdfsSource, InputContext, JdbcSource, Source
+from feathr.definition.transformation import (ExpressionTransformation, Transformation,
+                                   WindowAggTransformation)
+from feathr.definition.typed_key import TypedKey
+from feathr.registry.feature_registry import FeathrRegistry
+from feathr.definition.repo_definitions import RepoDefinitions
+from pathlib import Path
+from jinja2 import Template
+import sys
+from feathr.utils._file_utils import write_to_file
+import importlib
+import os
+
+class FeathrConfigHelper(object):
```

Comment:
Shall we add some unit test for this large config helper class?

Reply:
Yeah the focus of this PR is to restructure the code, the min bar is the E2E test should work, and we can add unit test in the follow up PR